Riot/Modules/Room/RoomInfo/RoomInfoList/RoomInfoListViewController.swift

```diff
@@ -127,25 +128,33 @@ final class RoomInfoListViewController: UIViewController {
         return self.theme.statusBarStyle
     }
     
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+
+        let height = ceil(basicInfoView.systemLayoutSizeFitting(CGSize(width: view.bounds.width, height: 0), withHorizontalFittingPriority: .required, verticalFittingPriority: .fittingSizeLevel).height)
+        
+        //  compare heights to avoid infinite loop
+        if height != basicInfoView.frame.height {
+            var headerFrame = basicInfoView.frame
+            headerFrame.size.height = height
+            basicInfoView.frame = headerFrame
+            mainTableView.tableHeaderView = basicInfoView
+        }
```

Comment:
Note: As `UITableView` do not manage Auto Layout for header and footer unfortunately we can factorize this work in a extension.

Reply:
Good idea, updated.